libs/domain/picking/src/components/picking-list-item/picking-list-item.component.ts

```diff
@@ -0,0 +1,113 @@
+import { resolve } from '@spryker-oryx/di';
+import { PickingListService } from '@spryker-oryx/picking';
+import { IconTypes } from '@spryker-oryx/themes/icons';
+import { Size } from '@spryker-oryx/ui';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import {
+  asyncState,
+  i18n,
+  isDefined,
+  observe,
+  valueType,
+} from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { BehaviorSubject, distinctUntilChanged, filter, switchMap } from 'rxjs';
+import { formatTime } from '../../utilities';
+import { styles } from './picking-list-item.styles';
+
+export class PickingListItemComponent extends LitElement {
+  static styles = styles;
+
+  protected pickingListService = resolve(PickingListService);
+
+  @property() pickingListId?: string;
+
+  @observe()
+  protected pickingListId$ = new BehaviorSubject(this.pickingListId);
+
+  // TODO: refactor this variable when start picking logic will be implemented (loading state should be the service area of responsibility)
+  @state()
+  protected isDisabled?: boolean;
+
+  protected pickingList$ = this.pickingListId$
+    .pipe(distinctUntilChanged())
+    .pipe(
+      filter(isDefined),
+      switchMap((pickingListId) =>
+        this.pickingListService.getById(pickingListId)
+      )
+    );
+
+  @asyncState()
+  pickingList = valueType(this.pickingList$);
+
+  protected startPicking(): void {
+    this.isDisabled = true;
+    // TODO: implement start piking logic
+
+    this.isDisabled = false;
+  }
+
+  protected showCustomerNote(): void {
+    if (!this.pickingList?.cartNote) {
+      return;
+    }
+
+    this.dispatchEvent(
+      new CustomEvent('oryx.show-note', {
+        detail: {
+          note: this.pickingList?.cartNote,
+        },
+      })
+    );
+  }
+
+  protected override render(): TemplateResult {
```

Comment:
```suggestion
  protected override render(): TemplateResult | void {
    if (!this.pickingList) return;
```
We could simplify some logic by adding this. no need to optional chaining and we can just use `<div class="time">${formatTime(this.pickingList.createdAt)}</div>`